modules/launcher/src/main/java/org/mule/module/launcher/artifact/DefaultResourceReleaser.java

```diff
@@ -32,6 +36,31 @@
     public void release()
     {
         deregisterJdbcDrivers();
+
+        cleanUpResourceBundle();
+    }
+
+    private void cleanUpResourceBundle()
+    {
+        try
+        {
+            final Method clearCacheMethod = ResourceBundle.class.getMethod("clearCache", ClassLoader.class);
```

Comment:
Commit message says "Uses reflection just in case if a Java version < 6" we know that that won't be the case on 3.x, 3.8.x and 3.7.x